New version: Polynomials4ML v0.3.1

JuliaRegistrator opened the pull request:
- Registering package: Polynomials4ML
- Repository: https://github.com/ACEsuit/Polynomials4ML.jl
- Created by: @cortner
- Version: v0.3.1
- Commit: 5615e0a6bfeb9868bdb84489369238b82a828b25
- Reviewed by: @cortner
- Reference: https://github.com/ACEsuit/Polynomials4ML.jl/issues/34#issuecomment-2184087829
- Description: Polynomials for ML: fast evaluation, batching, differentiation 
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598211c66e319c0b3944044738c0ae6d21f9aca015c947c5f72be651d0832fe371423cb7b6421ec0308ccf7d8440ddc459f37b4662a42b0fcabf56164a6f8dc1898a305e2bedbbacd7fddfea753062dadd95beafec8fe6e27d6a4b20b53114213c1c5145f30732224b5f57cc8376a1095097dbe4c6abc5902c752c22439d4cc64c4 -->

cortner commented:
I didn't notice that 0.3.0 was not merged. This PR can therefore be closed and I will first fix the 0.3.0 registration.

JuliaRegistrator committed:
New version: Polynomials4ML v0.3.1

UUID: 03c4bcba-a943-47e9-bfa1-b1661fc2974f
Repo: https://github.com/ACEsuit/Polynomials4ML.jl.git
Tree: 2b662884b0e89decc5de4c53b32ee3a7845d0383

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
